lib/CRS/Fuse/TS.pm

```diff
@@ -144,7 +144,12 @@ sub doFuseRepairMount {
 	die "ERROR: no Processing.Path.Repair specified!\n" unless defined $repairdir;
 
 	my $replacementpath = "$repairdir/$replacement";
-	return 0 unless -r $replacementpath;
```

Comment:
I don't know why, but `die "foo" unless -r $replacementpath` did neither print a log line nor set the preparing state to failed.

Reply:
Yes, die is in some way syntactically different, I also found it only to work with an if in front of it, but not with unless after it. I guess there is something like "parser stops reading once reaching die" or similar.